compiler/circle2circle-dredd-recipe-test/test.lst

```diff
@@ -61,6 +64,9 @@ Add(Net_Mul_Add_002 PASS remove_unnecessary_add)
 Add(Net_Mul_Add_003 PASS remove_unnecessary_add)
 Add(Net_Mul_Div_000 PASS fuse_mul_with_div)
 Add(Net_Mul_Div_001 PASS fuse_mul_with_div)
+Add(Net_Mul_FullyConnected_000 PASS fuse_mul_to_fullyconnected_weights fold_mul)
+Add(Net_Mul_FullyConnected_001 PASS fuse_mul_to_fullyconnected_weights fold_mul)
+Add(Net_Mul_FullyConnected_002 PASS fuse_mul_to_fullyconnected_weights fold_mul)
```

Comment:
Unused parameters..!?

```suggestion
Add(Net_Mul_FullyConnected_000 PASS fuse_mul_to_fullyconnected_weights)
Add(Net_Mul_FullyConnected_001 PASS fuse_mul_to_fullyconnected_weights)
Add(Net_Mul_FullyConnected_002 PASS fuse_mul_to_fullyconnected_weights)
```

Reply:
`fold_mul` is an option needed to constant fold `Mul` op,
- this model has `FC(Mul(input,scale), weights, bias)`.
- `fuse_mul_to_fullyconnected_weights` will convert to `FC(input, Mul(weights, scale), bias)`
- dredd test cannot detect the difference as there exist same number of FC and Mul
- `fold_mul` will convert to `FC(input, weights', bias)` and we can detect absence of `Mul`

Reply:
I understand.
Thank you for the description =)